.travis.yml

```diff
@@ -10,6 +10,8 @@ script: script/cibuild
 notifications:
   email: false
 before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update          ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install git     ; fi
```

Comment:
if you want to use multi line you can do it like this:
https://github.com/git/git/blob/master/.travis.yml#L37-L69